pkg/sql/plan/function/function_id.go

```diff
@@ -116,6 +118,7 @@ const (
 	TIMEDIFF
 	TIMESTAMPDIFF
 	DENSE_RANK
+	DIVISOR
```

Comment:
Are truncate and divisor well known SQL functions?   If only used in window, or internally, we should give them a better name, maybe prefix MO_WIN or something.

Reply:
done